app/src/main/java/com/example/poste/models/Post.java

```diff
@@ -104,6 +123,11 @@ public static class Builder {
          */
         private String id;
 
+        /**
+         * The list of tags as one comma separated string.
+         */
+        private String tags;
```

Comment:
Might be worth making a US (or working on the backend side-by-side in this US) to change over from a comma-separated string to an actual list, which would help alleviate some odd logic we have to use here in the front end

Reply:
Agreed
I think that is a point worth discussing next time we meet.